feature-libs/organization/account-summary/root/public_api.ts

```diff
@@ -0,0 +1,6 @@
+export * from './account-summary-root.module';
+export * from './config/index';
+export * from './facade/index';
+export * from './feature-name';
+export * from './model/account-summary.model';
+export * from './model';
```

Comment:
```
export * from './model/account-summary.model';
export * from './model';
```
can just be `export * from './model/index'`

Reply:
done